src/main.rs

```diff
@@ -2,17 +2,20 @@ mod connectivity;
 mod hotspot;
 mod platforms;
 
+use std::env::args;
 use crate::connectivity::{Connectivity, WifiConnectionError};
 use crate::platforms::{Config, WiFi};
 
+use std::process::{Command, Stdio};
+
 fn main() -> Result<(), WifiConnectionError> {
     let config = Some(Config {
-        interface: Some("wlo1"),
+        interface: Some("wlan0"),
     });
 
     let mut wifi = WiFi::new(config);
 
-    match wifi.connect("CSIS_MH", "") {
+    match wifi.connect("TP-Link_CEE2", "05Bs@sdfvn") {
         Ok(result) => println!(
             "{}",
             if result == true {

```

Comment:
Not sure these changes are needed. You can easily setup a test program that points to your own version of the changes.